flink-tests/src/test/java/org/apache/flink/test/runtime/ShuffleCompressionITCase.java

```diff
@@ -56,10 +57,12 @@
  */
 public class ShuffleCompressionITCase {
 
-	private static final int NUM_RECORDS_TO_SEND = 10 * 1024 * 1024;
+	private static final int NUM_RECORDS_TO_SEND = 1000 * 32 * 1024 / 12 + 1;
```

Comment:
It is better to give more explanations why we need to construct the number of records like this.

Reply:
I will add comment to explain that.